formats/protobuf-generator/jvmTest/resources/AbstractHolder.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto2";
+
+package kotlinx.serialization.protobuf.generator.scheme;
+
+// serial name: kotlinx.serialization.protobuf.generator.GenerationTest.AbstractHolder
+message AbstractHolder {
+  required Polymorphic_AbstractClass_ abs = 1;
+}
+
+// serial name: kotlinx.serialization.Polymorphic<AbstractClass>
+message Polymorphic_AbstractClass_ {
```

Comment:
It seems that such generator will produce new message for each polymorphic-base class. Is that what we intended?

Reply:
Ambiguous.
On the one hand we can reuse only one polymorphic message type. It gives shorter scheme.
On the other after protoc class generation the user will not be able to determine the type compatibility and can easily write data to the wrong type. Having an individual message, the user will be able to see the original polymorphic type and find it in the kotlin sources.